communication/slack-config/usergroups.yaml

```diff
@@ -224,3 +224,21 @@ usergroups:
       - rtaniwa
       - schultzp2020
       - yangcao77
+
+  - name: capi-release-team
```

Comment:
nit: I noticed the github team is called `cluster-api-release-team`, would it make sense to call them both the same so they're easier to find?